examples/tts/tts.cpp

```diff
@@ -452,6 +456,75 @@ static std::vector<llama_token> prepare_guide_tokens(const llama_vocab * vocab,
     return result;
 }
 
+static json speaker_from_file(const std::string & speaker_file) {
+    std::ifstream file(speaker_file);
+    if (!file) {
+        LOG_ERR("%s: Failed to open file '%s' for reading", __func__, speaker_file.c_str());
+        return json();
+    }
+
+    json speaker = json::parse(file);
+    return speaker;
+}
+
+static std::string get_tts_version(llama_model *model, json speaker = json::object()) {
```

Comment:
small nits here, maybe better if we use an enum for version number:

```
enum outetts_version {
    OUTETTS_V0_2,
    OUTETTS_V0_3,
};
```

Reply:
Thanks for the reminder! This part has been fixed.